dashboard/job-tracker.html

```
@@ -0,0 +1,1124 @@
+</head>
+<body>
+
+<!-- HEADER -->
+<div class="header">
+    <div class="header-top">
+        <div>
+            <h1>🚀 Hire_Help Job Search Command Center</h1>
+            <div class="header-subtitle">Data Analyst · Business Analyst · AI/ML Roles — Tracking everything in one place</div>
+        </div>
+        <div class="export-bar">
+            <button class="btn btn-secondary" onclick="exportCSV()">📥 Export CSV</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)